autotest/utils_tests.py

```diff
@@ -2570,8 +2570,73 @@ def thresh_pars_test():
     # plt.show()
 
 
+def test_ppu_import():
+    import pypestutils as ppu
+
+
+
+def ppu_geostats_test(tmp_path):
+    import sys
+    import os
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import pyemu
+    
+    import flopy
+
+    sys.path.insert(0,os.path.join("..","..","pypestutils"))
```

Comment:
Note for self: 
I think, shouldn't need this in the CI now -- we `pip install -e` the pypestutils directory directly so `import pypestutils as ppu` should "work". 

For a local install of ppu: `pip install -e {path to pypestutils setup.py}`

Reply:
I agree. That was just for me locally while we were waiting to a new ppu version to be released.  It's in several fxns so let's strip those out once things stabilize a bit more.